specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-03-01/orbital.json

```diff
@@ -0,0 +1,2228 @@
+        }
+      },
+      "description": "Tags object for patch operations."
+    },
+    "SpacecraftsProperties": {
+      "type": "object",
+      "required": [
+        "noradId"
+      ],
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ProvisioningState"
+            }
+          ],
+          "description": "The current state of the resource's creation, deletion, or modification"
+        },
+        "noradId": {
+          "type": "string",
+          "description": "NORAD ID of the spacecraft."
+        },
+        "titleLine": {
+          "type": "string",
+          "description": "Title line of Two Line Element (TLE)."
+        },
+        "tleLine1": {
+          "type": "string",
+          "description": "Line 1 of Two Line Element (TLE)."
+        },
+        "tleLine2": {
+          "type": "string",
+          "description": "Line 2 of Two Line Element (TLE)."
+        },
+        "links": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SpacecraftLink"
+          },
+          "x-ms-identifiers": [],
+          "description": "Links of the Spacecraft",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "description": "List of Spacecraft Resource Properties."
+    },
+    "SpacecraftListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Spacecraft"
+          },
+          "description": "A list of spacecraft resources in a resource group."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      },
+      "description": "Response for the ListSpacecrafts API service call."
+    },
+    "SpacecraftLink": {
+      "type": "object",
+      "required": [
+        "name",
+        "centerFrequencyMHz",
+        "bandwidthMHz",
+        "direction",
+        "polarization"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Link name",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "centerFrequencyMHz": {
+          "type": "number",
+          "description": "Center Frequency in MHz",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "bandwidthMHz": {
+          "type": "number",
+          "description": "Bandwidth in MHz",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "direction": {
+          "type": "string",
+          "description": "Direction (uplink or downlink)",
+          "enum": [
+            "uplink",
```

Comment:
Based on a live test, the enum of `direction` should be `Uplink` and `Downlink`.

Here're the requests traces: https://gist.github.com/ms-henglu/9f9d848881bad9c6347fc97543341cc1